src/FactSystem/FactMetaData.h

```diff
@@ -121,6 +121,7 @@ class FactMetaData : public QObject
     static const int defaultDecimalPlaces = 3;
 
     static ValueType_t stringToType(const QString& typeString, bool& unknownType);
+    static size_t typeToSize(ValueType_t);
```

Comment:
Yup, sorry I will make it momentarily!

On Wed, Feb 17, 2016 at 11:52 PM Don Gagne <notifications@github.com> wrote:

> In src/FactSystem/FactMetaData.h
> <https://github.com/mavlink/qgroundcontrol/pull/2821#discussion_r53271350>
> :
>
> > @@ -121,6 +121,7 @@ class FactMetaData : public QObject
> >      static const int defaultDecimalPlaces = 3;
> >
> >      static ValueType_t stringToType(const QString& typeString, bool& unknownType);
> > +    static size_t typeToSize(ValueType_t);
>
> @NaterGator <https://github.com/NaterGator> Can you make this one change
> and this should be good to go right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mavlink/qgroundcontrol/pull/2821/files#r53271350>.
>